pkg/storage/mvcc_value.go

```diff
@@ -234,6 +234,32 @@ func DecodeMVCCValue(buf []byte) (MVCCValue, error) {
 	return decodeExtendedMVCCValue(buf)
 }
 
+// DecodeValueFromMVCCValue decodes and MVCCValue and returns the
+// roachpb.Value portion without parsing the MVCCValueHeader.
+//
+// NB: Caller assumes that this function does not copy or re-allocate
+// the underlying byte slice.
+func DecodeValueFromMVCCValue(buf []byte) (roachpb.Value, error) {
```

Comment:
nit: could you add a unit test for this?

Reply:
Done. Piggybacked on the tests for DecodeMVCCValue so that we are always testing the same cases for both.

Reply:
Thanks!